modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v5_3_2/TemplateUpgradeProcess.java

```diff
@@ -23,28 +24,37 @@
 import java.sql.PreparedStatement;
 import java.sql.ResultSet;
 
+import java.util.ArrayList;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
 /**
  * @author Albert Gomes Cabral
+ * @author Felipe Veloso
  */
-public abstract class BaseTemplateUpgradeProcess extends UpgradeProcess {
+public class TemplateUpgradeProcess extends UpgradeProcess {
 
 	@Override
 	protected void doUpgrade() throws Exception {
-		_upgradeDDMTemplates();
-		_upgradeFragmentEntries();
-	}
-
-	protected String getTemplateContextVariable() {
-		return null;
+		ArrayList<Template> templates = new ArrayList<>();
+
+		templates.add(
+			new Template(
+				"\\w+\\s*\\=\\s*.+com\\.liferay\\.portal\\.kernel\\." +
+					"servlet\\.BrowserSnifferUtil\\\"\\)",
+				StringPool.BLANK, "browserSniffer"));
+
+		templates.add(
+			new Template(
+				"\\w+\\s*=\\s*serviceContext\\.getThemeDisplay\\(\\)",
+				StringPool.BLANK, "themeDisplay"));
```

Comment:
@velosofelipe64, if the `serviceContext.getThemeDisplay` declaration doesn't generate an issue in 7.4 then we can remove this part for now.